src/components/atoms/Button/Button.tsx

```diff
@@ -62,32 +62,36 @@ export const buttonStyle = css<ButtonProps>`
     `;
   }}
 
-  &:hover:not(:disabled) {
-    background: ${({ styling = 'primary' }) => colorMap[styling].hover};
-  }
-
   &:focus:not(:active) {
     border: 1px solid ${colors.white};
     box-shadow: 0 0 4px ${colors.actionPlain};
   }
 
-  &:disabled {
-    cursor: not-allowed;
-    ${({ loading }) => {
-      if (!loading) {
-        return css`
-          background: ${colors.greyLightest};
-          color: ${colors.grey};
-        `;
+  ${({ disabled, styling = 'primary' }) => {
```

Comment:
I'd suggest adding this example to the docs https://github.com/zopaUK/react-components/pull/616#issuecomment-710039789